app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    cmd = command.split()
```

Comment:
Before it, pls check that `len(cmd)==3`

Reply:
"mv" in cmd not correct check
because:
commad == "cp filemv.txt file2.txt"